src/routes/FavoritesScreen.tsx

```diff
@@ -0,0 +1,49 @@
+import { CommonActions, RouteProp, useFocusEffect } from '@react-navigation/native'
+import { StackNavigationProp } from '@react-navigation/stack'
+import React from 'react'
+
+import ServerResponseHandler from '../components/ServerResponseHandler'
+import VocabularyList from '../components/VocabularyList'
+import labels from '../constants/labels.json'
+import useLoadAsync from '../hooks/useLoadAsync'
+import { RoutesParams } from '../navigation/NavigationTypes'
+import AsyncStorage from '../services/AsyncStorage'
+
+interface FavoritesScreenProps {
+  route: RouteProp<RoutesParams, 'Favorites'>
+  navigation: StackNavigationProp<RoutesParams, 'Favorites'>
+}
+
+const FavoritesScreen = ({ navigation }: FavoritesScreenProps): JSX.Element => {
+  const { data, refresh, error } = useLoadAsync(AsyncStorage.getFavorites, {})
+
+  useFocusEffect(refresh)
+
+  const onItemPress = (index: number) => {
+    if (!data) {
+      return
+    }
+    navigation.navigate('VocabularyDetail', {
+      disciplineId: 0,
```

Comment:
VocabularyDetail route does not need disciplineId and disciplineTitle, does it? Should be removed.

Reply:
oke, fine for me. 

Not sure if it is bad practice to use "0", as this should be "null" or "undefined".

Reply:
removed it after all

Reply:
changed to null